### DISCUSSION TITLE
Welcome to Comet Discussions!

### DISCUSSION THREAD

**Discussion by Clancey** [dotnet/Comet#183] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by PySom** [comment]
I'm Chisom learning dotnet (C#) and I'll love to be a comet code contributor some day.

**Reply by Clancey** [reply]
I am excited for that day!

**Comment by qcjxberin** [comment]
Maui has a preview version now, not sure if there will be further development

**Reply by Clancey** [reply]
There most certainly will be more development here!

**Reply by qcjxberin** [reply]
Can there be more cases? Or the development of some commonly used small components

**Comment by alemputes** [comment]
Impressive work Clancey, really hope Comet gets official support and is released as a product. I work as a .NET developer and have done some XAML but never liked it, I totally agree with you, writing less code that is more readable is very valuable. I also try ed Flutter and was hooked but if I cloud use .NET on BackEnd + .NET MAUI + WEB + Comet on FrontEnd in production that would be mind-blowing :) Keep up the good work.

**Reply by tcsaddul** [reply]
The Comet concept is really good.  I hope MS will support it eventually.

**Comment by samcov** [comment]
Is this something like a new and improved **Silverlight**?

**Comet** adds targets that didn't exist during the Silverlight days, like **Android** & **Blazor**, and while **iOS** did exist, **Apple** was uncooperative, read anti-competitive :).

**Reply by saint4eva** [reply]
No.
